models/music_cont_out_proxy.h

```diff
@@ -0,0 +1,226 @@
+/*
+ *  music_cont_out_proxy.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef MUSIC_CONT_OUT_PROXY_H
+#define MUSIC_CONT_OUT_PROXY_H
+
+// Generated includes:
+#include "config.h"
+
+
+#ifdef HAVE_MUSIC
+
+// C includes:
+#include <mpi.h>
+
+// C++ includes:
+#include <vector>
+
+// External includes:
+#include <music.hh>
+
+// Includes from nestkernel:
+#include "nest_types.h"
+#include "node.h"
+
+// Includes from sli:
+#include "arraydatum.h"
+
+/* BeginDocumentation
+
+Name: music_cont_out_proxy - A device which sends continuous data from NEST to
+MUSIC.
+
+Description:
+A music_cont_out_proxy can be used to send continuous data from neurons over
+MUSIC to
+remote applications. It works in a similar like a multimeter model. The user
+has to specify the recordable values to observe (e.g. ["V_m"]) via the
+record_from parameter.
+The target neurons are specified by a list of global neuron ids which must be
+passed via
+the "targets" parameter. The music_cont_out_proxy will be connected
+automatically to the
+specified target neurons. It is not possible to change the list of target
+neurons or observed
+quantities once they have been set or the simulation has been started for the
```

Comment:
Currently, these lists can be set as often as wished. But not after `Simulate` has been called (which is the correct behavior in my view). The documentation needs to reflect this behavior.